src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertificate.swift

```diff
@@ -280,6 +284,19 @@ final class HealthCertificate: Codable, Equatable, Comparable, RecycleBinIdentif
 			return hashData == $0.hash
 		}
 	}
+	
+	func belongsToSamePerson(_ other: HealthCertificate) -> Bool {
+		let hasSameDateOfBirth = self.trimmedDateOfBirth == other.trimmedDateOfBirth
```

Comment:
As a performance optimization this could be a guard and we could only do the other operations if this is true. Just an idea!

Reply:
done 🏎💨